webapp/src/assetEditor.tsx

```diff
@@ -16,11 +16,26 @@ function init() {
 }
 
 export class AssetEditor extends React.Component {
+    private editor: FieldEditorComponent<any>;
+
+    handleMessage = (msg: any)  => {
+        if (msg.data._fromVscode && msg.data.type === "update") {
+            this.editor.loadJres(msg.data.message);
+        }
+    }
+
+    refHandler = (e: FieldEditorComponent<any>) => {
+        this.editor = e;
+    }
+
+    componentDidMount() {
+        window.onmessage = this.handleMessage;
```

Comment:
best to use ``window.addEventHandler("message", this.handleMessage, null)`` and remove handler in unmount